uwp/Visual-Studio-Integration/NuGet-Packages.md

```diff
@@ -25,13 +25,13 @@ The NuGet Package Manager can be used to search and install NuGet packages in th
 
 1.	Right-click the project or solution in the Solution Explorer tab, and choose **Manage NuGet Packages…**
 
-    ![NuGet package manager add-in for windows](NuGet_Packages_Images/img3.png)
+    ![UWP Visual Studio Integration NuGet package manager add-in for windows](nuget-packages-images/uwp-visual-studio-integration-nuget-packages-installation.png)
```

Comment:
Image Folder Name seems wrong.
Check and update.
<img width="931" alt="image" src="https://github.com/syncfusion-content/uwp-docs/assets/44014283/16842747-0b28-44e6-8876-14eccadc4577">

Reply:
resolved